src/main/java/xyz/vegaone/easytracking/service/SprintService.java

```diff
@@ -0,0 +1,81 @@
+package xyz.vegaone.easytracking.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import xyz.vegaone.easytracking.domain.SprintEntity;
+import xyz.vegaone.easytracking.dto.Sprint;
+import xyz.vegaone.easytracking.mapper.SprintMapper;
+import xyz.vegaone.easytracking.mapper.UserStoryMapper;
+import xyz.vegaone.easytracking.repo.SprintRepo;
+import xyz.vegaone.easytracking.repo.UserStoryRepo;
+
+import java.util.Collections;
+import java.util.Optional;
+
+@Service
+public class SprintService {
+
+    private SprintRepo sprintRepo;
+
+    private SprintMapper sprintMapper;
+
+    private UserStoryService userStoryService;
+
+    private UserStoryRepo userStoryRepo;
```

Comment:
`userStoryRepo` and `userStoryMapper` are not used. They can be safely removed. Don't forget to remove them from the constructor as well.